components/AudioPlayer/AudioPlayer.tsx

```diff
@@ -172,6 +172,7 @@ export const AudioPlayer = ({
         <div ref={rootElm} className={rootClasses}>
           <IconButton
             className={playBtnClasses}
+            aria-label="Play"
```

Comment:
This should probably update based on player state: `aria-label={playing ? 'Pause' : 'Play'}`

Reply:
Fixed!